docker-compose.yml

```diff
@@ -17,7 +17,9 @@ services:
       - mysql
     volumes:
       - ${PWD}:/opt/coding-challenge-api
-    command: /bin/bash -c "sleep 25; rake db:drop db:create db:migrate && rails server -b 0.0.0.0"
+    command: /bin/bash -c "sleep 25; rake db:drop db:create db:migrate && rm tmp/pids/server.pid && rails server -b 0.0.0.0"
+    tty: true
+    stdin_open: true
```

Comment:
These two last configuration lines make it possible to debug a running container.

We can run in a terminal the application with `docker-compose up`.

Then we can find the container ID executing the rails server with `docker ps`. 

Having the container ID we can attach to it by executing `attach [container id]`.  If the code has a break point somewhere, this attachment will leave us to use it.